101.md

```diff
@@ -0,0 +1,42 @@
+NIP-101
+======
+
+Standard HTTP REST API for Relays
+-------------------------------
+
+`draft` `optional` `author:jacany`
+
+This NIP defines the structure for a standard REST API to be implemented by relays. This is to ensure compatibility between different relay implementations and the clients accessing them.
+
+## Reading Events
+
+GET `/api/event`
+
+Tags should be passed as query paramaters
+`/api/req?ids=id1,id2,id3&authors=pk1&since=0`
```

Comment:
```suggestion
`/api/event?ids=id1,id2,id3&authors=pk1&since=0`
```

Reply:
Any specific reason for this?

Reply:
> GET `/api/event`

Isn't this a REST API endpoint?

Reply:
Yes? Though req is used in the websocket for subscribing unless i'm mistaken...